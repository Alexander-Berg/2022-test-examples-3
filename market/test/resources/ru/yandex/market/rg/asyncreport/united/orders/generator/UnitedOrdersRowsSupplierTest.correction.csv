SHOPS_WEB.ENVIRONMENT
NAME,VALUE
mbi.control.enabled.reports,true
db.order.service.use_partner_payout_transactions_query,true

SHOPS_WEB.PARTNER
ID,TYPE
501,SUPPLIER
10614662,BUSINESS
431782,SHOP

SHOPS_WEB.PARTNER_PLACEMENT_PROGRAM
PARTNER_ID,PROGRAM,STATUS
431782,DROPSHIP,SUCCESS
501,FULFILLMENT,SUCCESS

SHOPS_WEB.BUSINESS
ID,NAME,CAMPAIGN_ID,DELETED_AT
10614662,business,1,null

SHOPS_WEB.DATASOURCE
ID,MANAGER_ID
431782,-2

SHOPS_WEB.SUPPLIER
ID,NAME
501,someShop

MARKET_BILLING.CAMPAIGN_INFO
DATASOURCE_ID,CAMPAIGN_ID,START_DATE
501,501,2020-01-01

SHOPS_WEB.BUSINESS_SERVICE
BUSINESS_ID,SERVICE_ID,SERVICE_TYPE
10614662,431782,SHOP
10614662,501,SUPPLIER

SHOPS_WEB.PARTNER_APP_META
ID,CREATION_TIME,TYPE,UPDATE_DATE
1201,2016-01-01,MARKETPLACE,2018-05-06 12:30:00
62,2016-01-01,MARKETPLACE,2018-05-06 12:30:00

SHOPS_WEB.PARTNER_APP
REQUEST_ID,PARTNER_ID,STATUS
1201,501,COMPLETED
62,431782,COMPLETED

SHOPS_WEB.PARTNER_APP_BUSINESS
DATASOURCE_ID,REQUEST_ID
501,1201
431782,62

SHOPS_WEB.PARTNER_APP_CONTACT
DATASOURCE_ID,REQUEST_ID,CONTACT_PERSON,PHONE_NUMBER,EMAIL
501,1201,1,1,1
431782,62,1,1,1

SHOPS_WEB.PARTNER_APP_BALANCE_DATA
REQUEST_ID,SELLER_CLIENT_ID
1201,10000
62,20000

SHOPS_WEB.SUPPLIER_CONTRACT
SUPPLIER_ID,REQUEST_ID,CLIENT_ID,PERSON_ID,CONTRACT_ID,CONTRACT_TYPE,CURRENCY
501,1201,777,13,500001,income,RUR
431782,62,778,14,500002,income,RUR

MARKET_BILLING.PAYOUT
PAYOUT_ID,ENTITY_ID,CHECKOUTER_ID,TRANSACTION_TYPE,PRODUCT,PAYSYS_TYPE_CC,ORDER_ID,PARTNER_ID,TRANTIME,AMOUNT,PAYOUT_GROUP_ID,ENTITY_TYPE

MARKET_BILLING.PAYOUT_CORRECTION
PAYOUT_ID,ENTITY_ID,CHECKOUTER_ID,TRANSACTION_TYPE,PRODUCT,PAYSYS_TYPE_CC,ORDER_ID,PARTNER_ID,TRANTIME,AMOUNT,UUID,LOGIN,ST_TICKET,PAYOUT_GROUP_ID,ENTITY_TYPE,PAYSYS_PARTNER_ID,ACCRUAL_CORRECTION_ID

MARKET_BILLING.ACCRUAL
ENTITY_ID,CHECKOUTER_ID,TRANSACTION_TYPE,PRODUCT,PAYSYS_TYPE_CC,ORDER_ID,PARTNER_ID,TRANTIME,AMOUNT,EXPORTED_TO_TLOG,ENTITY_TYPE,ORG_ID,CURRENCY,PAYOUT_STATUS

MARKET_BILLING.ACCRUAL_CORRECTION
ID,ENTITY_ID,PRODUCT,PAYSYS_TYPE_CC,CHECKOUTER_ID,TRANSACTION_TYPE,ORDER_ID,PARTNER_ID,TRANTIME,AMOUNT,EXPORTED_TO_TLOG,ENTITY_TYPE,PAYSYS_PARTNER_ID,CURRENCY,ORG_ID,UUID,LOGIN,ST_TICKET

MARKET_BILLING.PAYMENT_ORDER
ID,CLIENT_ID,CONTRACT_ID,SERVICE_ID,TRANSACTION_TYPE,SECURED_PAYMENT,FACTORING,PRODUCT,PAYSYS_TYPE_CC,TRANTIME,AMOUNT,CURRENCY,EXPORTED_TO_TLOG

MARKET_BILLING.PAYOUT_GROUP_PAYMENT_ORDER
PAYMENT_ORDER_ID,PAYOUT_GROUP_ID

MARKET_BILLING.CPA_ORDER
ORDER_ID,SHOP_ID,CREATION_DATE,SHOP_CURRENCY,FREE,TRANTIME,BILLING_STATUS,STATUS,FEE_SUM,DELIVERY,ITEMS_TOTAL,CAMPAIGN_ID,PAYMENT_TYPE,DELIVERY_TRANTIME,REGION_TO,COLOR,PAYMENT_METHOD,PAYMENT_SUBMETHOD
47375313,431782,2021-05-19 21:52:01,RUR,null,2021-05-25,1,5,0,0,1273500,21421814,1,2021-05-25,42,1,4,null

MARKET_BILLING.CPA_ORDER_ITEM
ID,ORDER_ID,ORDER_STATUS,OFFER_NAME,PRICE,BUYER_PRICE,SUBSIDY,ITEM_COUNT,FF_SUPPLIER_ID,FF_SHOP_ID,SHOP_SKU,FEED_ID,OFFER_ID,WARE_MD5,CAT_ID,FEED_CAT_ID,SHOW_UID,FEE_NORM,FEE_INT,FEE_UE,FEE_NET_UE,WAREHOUSE_ID,INITIAL_COUNT
90601894,47375313,5,Льдогенератор Gorenje IMC1200B,1273500,1273500,161500.00,1,501,null,1054608,475690,660546.1054608,UemIy2jddlmkFykQZwOzsA,16657217,null,162142849020825491469,0,0,0,0,47828,1

MARKET_BILLING.CPA_ORDER_TRANSACTION
ID,SHOP_ID,BANK_ORDER_ID,TRUST_REFUND_ID,TRAN_SUM,STATUS,TRANTIME,EVENTTIME,CLIENT_TYPE,TRUST_PAYMENT_ID,TYPE,BANK_ORDER_TIME,PAYMENT_ID,REFUND_ID,CREATED_AT,MBI_CONTROL_ENABLED,CESSION
36696011,431782,null,null,161500,1,2021-05-25 18:17:28.000000,2021-05-25 18:17:28,1,60ad1507f9880147d1ae5102,1,null,54700892,null,2021-05-25 18:17:26.000000,null,null
36696015,431782,null,null,1273500,2,2021-05-25 18:17:28.000000,2021-05-25 18:17:28,1,60ad150d94d52734ddd385b7,2,null,54700902,null,2021-05-25 18:17:32.000000,null,null
57593275,431782,null,6195f3dc2fe0987bc34fa9c1,1273500,0,2021-11-18 09:34:03.000000,2021-11-18 09:34:03,0,null,2,null,null,7149243,2021-11-18 09:34:03.000000,0,0
57593350,431782,null,6195f4107153b3a067674158,161500,2,2021-11-18 09:37:43.000000,2021-11-18 09:34:03,0,null,1,null,null,7149244,2021-11-18 09:34:03.000000,0,0

MARKET_BILLING.ORDERS_TRANSACTIONS
ORDER_ID,TRANSACTION_ID
47375313,36696011
47375313,36696015
47375313,57593275
47375313,57593350

MARKET_BILLING.ORDER_TRANTIMES
ORDER_ID,SERVICE_TYPE,TRANTIME
47375313,ff_processing,2021-05-22 21:37:08.406000
47375313,agency_commission,2021-05-25 18:17:35.000000
47375313,delivery_to_customer,2021-05-22 23:51:09.727000
47375313,fee,2021-05-25 18:17:47.090000

MARKET_BILLING.BANK_ORDER
PAYMENT_BATCH_ID,TRANTIME,EVENTTIME,STATUS,BANK_ORDER_ID,SUM,OEBS_STATUS,SERVICE_ID
265241547,2021-05-27 10:34:57.000000,2021-05-26 00:00:00.000000,done,44729,69983200,reconciled,610
251423649,2021-08-02 11:09:17.000000,2021-07-30 00:00:00.000000,done,775983,873589532,reconciled,609
378590716,2021-12-02 12:23:49.000000,2021-12-01 00:00:00.000000,done,24604,-161500,reconciled,609
376971163,2021-12-02 12:33:10.000000,2021-12-01 00:00:00.000000,done,19316,1178863116,reconciled,6101

MARKET_BILLING.BANK_ORDER_ITEM
PAYMENT_BATCH_ID,TRANSACTION_TYPE,TRUST_ID,SERVICE_ORDER_ID,ORDER_ID,ORDER_ITEM_ID,SUM,CURRENCY,PAYMENT_TYPE,HANDLING_TIME,PAYMENT_TIME,CONTRACT_ID,PARTNER_ID,AGENCY_COMMISSION,PAYMENT_ORDER_ID
265241547,payment,60ad150d94d52734ddd385b7,47375313-item-90601894,47375313,90601894,1273500,RUR,cash,2021-05-25 18:17:35.000000,2021-05-25 18:17:35.000000,363226,501,0,null
251423649,payment,60ad1507f9880147d1ae5102,47375313-item-subsidy-90601894,47375313,90601894,161500,RUR,subsidy,2021-05-25 18:17:28.000000,2021-05-25 18:17:28.000000,363227,501,0,null
378590716,refund,6195f4107153b3a067674158,47375313-item-subsidy-90601894,47375313,90601894,161500,RUR,subsidy,2021-11-18 09:34:56.000000,2021-11-18 09:34:56.000000,363227,501,0,null
376971163,refund,6195f3dc2fe0987bc34fa9c1,47375313-cash-7149243-refund-90601894-1,47375313,90601894,1273500,RUR,cash,2021-11-18 09:34:04.000000,2021-11-18 09:34:04.000000,363226,501,0,null

MARKET_BILLING.RECEIPT_ITEM
ITEM_ID,PAYMENT_ID,REFUND_ID,COUNT,PRICE,AMOUNT,SPASIBO,ORDER_ID
90601894,54700892,null,1,1615.00,1615.00,null,47375313
90601894,54700902,null,1,12735.00,12735.00,null,47375313
90601894,null,7149244,1,1615.00,1615.00,null,47375313
90601894,null,7149243,1,12735.00,12735.00,null,47375313

SHOPS_WEB.DELIVERY_SERVICES
ID,NAME,HUMAN_READABLE_ID,PICKUP_AVAILABLE,IS_COMMON,MARKET_STATUS,TYPE
145,Маршрут ФФ,MarschrouteFF,0,0,on,fulfillment

SHOPS_WEB.PARTNER_FF_SERVICE_LINK
PARTNER_ID,SERVICE_ID,FEED_ID
501,145,475690

MARKET_BILLING.CPA_ORDER_STATUS_HISTORY
ORDER_ID,STATUS,TRANTIME
47375313,0,2021-05-19 21:52:04.116000
47375313,1,2021-05-22 23:51:09.727000
47375313,5,2021-05-25 18:17:47.090000
47375313,7,2021-05-19 21:52:03.267000
47375313,6,2021-05-25 09:47:26.459000

MARKET_BILLING.ACCRUAL_TRANTIME
ID,TRANTIME_TYPE,TRANTIME,PROCESSED

