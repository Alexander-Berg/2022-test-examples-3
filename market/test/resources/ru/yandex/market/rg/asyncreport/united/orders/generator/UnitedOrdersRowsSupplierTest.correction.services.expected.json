[
    {
        "header": {
            "businessId": 10614662,
            "partnerPlacementProgramType": "FULFILLMENT",
            "partnerId": 501,
            "partnerName": "someShop",
            "inn": null,
            "placementContract": null,
            "promotionContract": null,
            "partnerNameAndPlacement": "someShop FBY"
        },
        "orderId": 47375313,
        "orderNum": null,
        "orderCreationDate": [
            2021,
            5,
            19
        ],
        "orderStatus": "Доставлен",
        "all": 0.00,
        "incomeWithSubsidy": 14350.0000,
        "incomeWithoutServices": 14350.0000,
        "incomeWithoutSubsidy": 12735.00,
        "paymentStatus": "Платеж переведен",
        "paymentBatchId": "44729",
        "saleCommission": 0.00,
        "warehouseHandling": 0.00,
        "installment": 0.00,
        "delivery": 0.00,
        "expressDelivery": 0.00,
        "transferPayment": 0.00,
        "sortCenterHandling": 0.00,
        "resupplyHandling": 0.00,
        "returnResupply": 0.00,
        "loyaltyProgram": 0.00,
        "cpaAuctionPromotion": 0.00
    }
]
